database/sql/src/main/scala/cromwell/database/sql/SqlConverters.scala

```diff
@@ -30,11 +30,17 @@ object SqlConverters {
   }
 
   implicit class StringOptionToClobOption(val strOption: Option[String]) extends AnyVal {
-    def toClob: Option[Clob] = strOption.flatMap(_.toClob)
+    def toClob: Option[Clob] = strOption.flatMap(_.toClobOption)
   }
 
   implicit class StringToClobOption(val str: String) extends AnyVal {
-    def toClob: Option[Clob] = if (str.isEmpty) None else Option(new SerialClob(str.toCharArray))
+    def toClobOption: Option[Clob] = if (str.isEmpty) None else Option(new SerialClob(str.toCharArray))
+
+    def toClob(default: String): Clob = {
+      require(default.nonEmpty, "An empty string has been passed in where a non-empty default string is required to convert String objects to Clob.")
```

Comment:
Is this where people were asking for the `String Refined NonEmpty`? `require` will just throw an exception otherwise IIRC